runtime/onert/core/src/compiler/OperationValidator.cc

```diff
@@ -75,6 +75,15 @@ void OperationValidator::visit(const ir::operation::AvgPool2D &node)
   OP_REQUIRES(_ctx.at(ifm_index).shape().rank() == 4);
 }
 
+void OperationValidator::visit(const ir::operation::BatchMatMul &node)
+{
+  const auto lhs_index(node.getInputs().at(ir::operation::BatchMatMul::Input::LHS));
+  const auto rhs_index(node.getInputs().at(ir::operation::BatchMatMul::Input::RHS));
+
+  // Constant lhs and rhs is not implemented yet
+  OP_REQUIRES(!_ctx.at(lhs_index).isConstant() && !_ctx.at(rhs_index).isConstant());
```

Comment:
@hseok-oh I cannot find some of the validation check done in tensorflow lite `Prepare`. e.g. 

```c++
  // Support dimensions between 2 and 4, inclusive.
  TF_LITE_ENSURE(context, NumDimensions(lhs_data) >= 2);
  TF_LITE_ENSURE(context, NumDimensions(lhs_data) <= 4);
  TF_LITE_ENSURE(context, NumDimensions(rhs_data) >= 2);
  TF_LITE_ENSURE(context, NumDimensions(rhs_data) <= 4);
```

Reply:
I didn't check dimension check because of dynamic shape (our example model's input is scalar - unknown)

Reply:
@hseok-oh I met similar situation during reviewing shape inference. Someone would like to remove existing check. However I prefer to update this check with considering unknown. For example, we may write sth like:

`lhs.hasUnspecified() || NumDimensions(lhs) >= 2`.